lib/components/app/desktop-nav.tsx

```diff
@@ -55,9 +96,28 @@ const DesktopNav = ({ otpConfig, popupTarget, setPopupContent }: Props) => {
               className={branding && `with-icon icon-${branding}`}
               style={{ marginLeft: 50 }}
             >
-              {/* A title is always rendered (e.g.for screen readers)
-                  but is visually-hidden if a branding icon is used. */}
               <div className="navbar-title">{title}</div>
+              {logoClickAction === 'start-over' && (
+                <TransparentButton bsStyle="link" onClick={handleStartOver}>
+                  <InvisibleA11yLabel>
+                    <FormattedMessage id="components.AppMenu.agencyLogoReset" />
+                  </InvisibleA11yLabel>
+                </TransparentButton>
+              )}
+              {logoClickAction?.startsWith('http') && (
+                <TransparentButton
+                  bsStyle="link"
```

Comment:
Just realizing that these buttons technically work as links, which means for a11y purposes the `TransparentButton` component needs to be changed to an anchor tag (this would be ideal!), or these two buttons need to have a `role="link"`!